front/components/assistant_builder/ActionsScreen.tsx

```diff
@@ -543,179 +653,64 @@ function ActionEditor({
   ].includes(action.type as any);
 
   const shouldDisplayActionDescription = action.type !== "DUST_APP_RUN";
+  const shouldDisplayAdvancedSettings = action.type !== "DUST_APP_RUN";
```

Comment:
I guess we could just do one of those?

Reply:
Yeah I think you're right these 2 will stay aligned 👍